### PR TITLE
Update change logs

### PR DESCRIPTION
**Pull request by abdulhaseeb2**

### CODE REVIEW

**Comment by stakater-user** [comment]
@abdulhaseeb2 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-329-a36b24b7`

File: content/sre/tenant-operator/changelog.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## v0.5.0
+
+### Changes
+
+- feat: Add support for tenant namespaces off-boarding. For more details check out [onDelete](./customresources.html#_2-tenant)
+- feat: Add tenant webhook for spec validation
+
+- fix: TemplateGroupInstance now removes created resources if TGI selector namespace label is updated
```
Comment (MuneebAijaz) [inline content/sre/tenant-operator/changelog.md]: `TemplateGroupInstance now cleans up leftover Template resources from namespaces that are no longer part of TGI namespace selector`

Comment (abdulhaseeb2) [reply content/sre/tenant-operator/changelog.md]: updated

**Review by MuneebAijaz — APPROVED** [review]

**Comment by stakater-user** [comment]
@abdulhaseeb2 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-329-c2c8441b`

**Comment by stakater-user** [comment]
@abdulhaseeb2 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:v0.0.255`

**Pull request merged**